exercises/proverb/.meta/src/reference/java/Proverb.java

```diff
@@ -0,0 +1,27 @@
+final class Proverb {
```

Comment:
This class doesn't really seem to follow the [canonical-data](https://github.com/exercism/problem-specifications/blob/master/exercises/proverb/canonical-data.json). It should just have one method, `recite`, which when given a list of words (you can give the list of words in the constructor if you want) outputs the proverb. It should not have, and should not need, a `setQualifier` method. Does that make sense? :)

Reply:
Just to clatify, you can name the method something else if you like, for example `getProverb`, but I think it's very important that it's only one method without any `setQualifier` method :)